### DISCUSSION TITLE
Force the version

### DISCUSSION THREAD

**Discussion by artbandin** [semantic-release/semantic-release#2988] [Configuration]
How can I force the version to be set for the next semantic-release tag?

We have been using CalVer for versioning our product. However, we decided to switch to semantic-release. As a result, the versions follow this pattern: 2023.Х.Х

We want to remove the year from the version and switch to the format 1.X.X. We tried manually creating a tag 1.X.X, but semantic-release still continued to set versions with 2023.

How can we solve this issue?

**Comment by travi** [comment]
https://semantic-release.gitbook.io/semantic-release/usage/configuration#existing-version-tags

**Reply by artbandin** [reply]
Unfortunately, this does not help me solve the problem. I understand how semantic-release determines which commits to include in the next version. However, it is not clear from the mentioned article how I can forcefully change the number of the future version.

**Reply by gr2m** [reply]
It's not possible, and goes against the nature of semantic-release. You can create a tag after a release happened, and that will be taken as base for calculating the next version

**Reply by artbandin** [reply]
I tried manually creating a tag twice after the release, but semantic-release ignored it during the next release:
https://take.ms/PIeKv

**Reply by gr2m** [reply]
Hm odd, I did it in the past and it worked. Sorry I can't help debug this further from my side right now. I'd look into the logs next maybe it has hints why the lower version was taken as a base

**Reply by artbandin** [reply]
It seems that the problem is actually that the base is taken from the higher version — 2023.X.X (CalVer, which was used previously). But I need the base of the new release to be taken from the lower version 1.X.X, which I am manually creating. And I don't understand how to do this 😔
